crates/astria-sequencer/src/app/mod.rs

```diff
@@ -602,7 +602,7 @@ impl App {
         txs: Vec<SignedTransaction>,
         block_size_constraints: &mut BlockSizeConstraints,
     ) -> anyhow::Result<()> {
-        let mut excluded_tx_count = 0;
+        let mut excluded_tx_count = 0.0_f64;
```

Comment:
Extremely nit
```suggestion
        let mut excluded_tx_count = 0_f64;
```

Reply:
Done in [44c99af](https://github.com/astriaorg/astria/pull/1081/commits/44c99affc52d66ee3134fa1bbc23f66d21bb632a).